tests/yam/validate/validate_base_product.pm

```diff
@@ -0,0 +1,23 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Validate the base product from /etc/products.d/*.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use Test::Assert ':assert';
+
+sub run {
+    record_info("baseproduct", "Validate base product");
```

Comment:
ahm this wasn't needed when using a test module instead of a function. Please drop.

Reply:
done, thanks.